src/schedule/components/MyScheduleContent.tsx

```diff
@@ -0,0 +1,68 @@
+// 내 일정 목록
+import {
+  defaultCardList,
+  defaultParticipateList,
+  defaultTemporaryCardList,
+} from "@schedule/const";
+import React, { useState } from "react";
+import ScheduleCard, { cardType } from "./ScheduleCard";
+import { CardItemType, scheduleType } from "./ScheduleContent";
+import TitleCardContainer from "./TitleCardContainer";
+
+const MyScheduleContent = () => {
+  const [onComingCardList, setOnComingCardList] =
+    useState<CardItemType[]>(defaultCardList);
+  const [temporaryCardList, setTemporarayCardList] = useState<CardItemType[]>(
```

Comment:
사소한 부분이지만 오타가 있습니다ㅎㅎ
```suggestion
  const [temporaryCardList, setTemporaryCardList] = useState<CardItemType[]>(
```